ql/src/java/org/apache/hadoop/hive/ql/ddl/table/partition/drop/AlterTableDropPartitionOperation.java

```diff
@@ -120,6 +126,12 @@ private void dropPartitions() throws HiveException {
     List<Partition> droppedPartitions = context.getDb().dropPartitions(tablenName.getDb(), tablenName.getTable(),
         partitionExpressions, options);
 
+    if (isRepl) {
```

Comment:
what is this check

Reply:
It does a lot of stuff below related to ``llap`` and printing to console, which I thought would be irrelevant here, so I added this check, So in case of replication, I skip the below part and return from here itself